python/lsst/ts/athexapod/csc.py

```diff
@@ -203,7 +203,9 @@ async def end_start(self, data):
         """
         try:
             await self.controller.connect()
-            # TODO Add make_ackcmd call when it is released.
```

Comment:
I know the TODO is included in the `end_start` method, but it would actually be more suitable to have it in `begin_start` instead.  Or, if you don't want to add a `begin_start` method, call this at the top of the method instead of after calling the `self.controller.connect()` method. 

Also, instead of using a hard coded timeout. Consider using `self.controller.long_timeout` instead, since this is what the `connect` method uses as timeout.